lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -338,7 +338,11 @@ export default function FilePickerApp({ onSubmit }: FilePickerAppProps) {
         ref={formRef}
       >
         {isEditing && (
-          <RouterLink href="/app/basic-lti-launch" data-testid="back-link">
+          <RouterLink
+            href="/app/basic-lti-launch"
+            data-testid="back-link"
+            asChild
```

Comment:
`asChild` is now needed when nesting our own Link components into wouter's ones. Not providing it results in an anchor nested in another anchor.

See the `Link composition with asChild prop` entry in the release notes https://github.com/molefrog/wouter/releases/tag/v3.0.0